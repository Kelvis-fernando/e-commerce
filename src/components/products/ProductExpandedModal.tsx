import { ProductExpandedModalContainer } from "../../styles/products/productExpandedModal";

interface ProductExpandedProps {
  product: any;
}

const ProductExpandedModal = ({ product }: ProductExpandedProps) => {
  console.log(product);
  return (
    <ProductExpandedModalContainer>
      <div>
        <h1>{product?.title}</h1>
        <div>{product?.brand}</div>
        <div>{product?.price}</div>
      </div>
      <img src={product?.image} alt="product" />
    </ProductExpandedModalContainer>
  );
};

export default ProductExpandedModal;
