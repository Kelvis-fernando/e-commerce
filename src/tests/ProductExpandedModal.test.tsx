import React from "react";
import { render } from "@testing-library/react";
import ProductExpandedModal from "../components/products/ProductExpandedModal";

describe("ProductExpandedModal", () => {
  test("should render and match snapshot", () => {
    const { container } = render(<ProductExpandedModal />);

    expect(container).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
