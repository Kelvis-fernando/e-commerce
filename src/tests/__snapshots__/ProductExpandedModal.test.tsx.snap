// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductExpandedModal should render and match snapshot 1`] = `
<div>
  <div
    class="sc-bcXHqe iEWIbF"
  >
    <div
      class="sc-gswNZR jeIENF"
    >
      <h1>
        Sobre o café
      </h1>
      <h2 />
      <span>
        undefined $ undefined
      </span>
      <div
        class="sc-dkrFOg jIzRzQ"
      >
        <div>
          <div>
            <svg
              fill="currentColor"
              height="32"
              viewBox="0 0 256 256"
              width="32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <rect
                fill="none"
                height="256"
                width="256"
              />
              <path
                d="M224,177.3V78.7a8.1,8.1,0,0,0-4.1-7l-88-49.5a7.8,7.8,0,0,0-7.8,0l-88,49.5a8.1,8.1,0,0,0-4.1,7v98.6a8.1,8.1,0,0,0,4.1,7l88,49.5a7.8,7.8,0,0,0,7.8,0l88-49.5A8.1,8.1,0,0,0,224,177.3Z"
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
              />
              <polyline
                fill="none"
                points="177 152.5 177 100.5 80 47"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
              />
              <polyline
                fill="none"
                points="222.9 74.6 128.9 128 33.1 74.6"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
              />
              <line
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
                x1="128.9"
                x2="128"
                y1="128"
                y2="234.8"
              />
            </svg>
          </div>
          <h2>
            Classificação
          </h2>
          <h3 />
        </div>
        <div>
          <div>
            <svg
              fill="currentColor"
              height="32"
              viewBox="0 0 256 256"
              width="32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <rect
                fill="none"
                height="256"
                width="256"
              />
              <line
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
                x1="216"
                x2="40"
                y1="216"
                y2="216"
              />
              <rect
                fill="none"
                height="56"
                rx="8"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
                transform="translate(44 300) rotate(-90)"
                width="112"
                x="116"
                y="100"
              />
              <rect
                fill="none"
                height="152"
                rx="8"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
                width="56"
                x="56"
                y="32"
              />
            </svg>
          </div>
          <h2>
            Intensidade
          </h2>
          <h3 />
        </div>
        <div>
          <div>
            <svg
              fill="currentColor"
              height="32"
              viewBox="0 0 256 256"
              width="32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <rect
                fill="none"
                height="256"
                width="256"
              />
              <line
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
                x1="88"
                x2="88"
                y1="24"
                y2="56"
              />
              <line
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
                x1="120"
                x2="120"
                y1="24"
                y2="56"
              />
              <line
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
                x1="152"
                x2="152"
                y1="24"
                y2="56"
              />
              <line
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
                x1="32"
                x2="208"
                y1="216"
                y2="216"
              />
              <path
                d="M83.3,216A88,88,0,0,1,32,136V88H208v48a88,88,0,0,1-51.3,80"
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
              />
              <path
                d="M208,88h0a32,32,0,0,1,32,32v8a32,32,0,0,1-32,32h-3.4"
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
              />
            </svg>
          </div>
          <h2>
            Notas
          </h2>
          <h3 />
        </div>
        <div>
          <div>
            <svg
              fill="currentColor"
              height="32"
              viewBox="0 0 256 256"
              width="32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <rect
                fill="none"
                height="256"
                width="256"
              />
              <line
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
                x1="128"
                x2="128"
                y1="232"
                y2="88"
              />
              <line
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
                x1="128"
                x2="80"
                y1="152"
                y2="128"
              />
              <line
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
                x1="128"
                x2="176"
                y1="128"
                y2="104"
              />
              <path
                d="M132.5,181.2A64,64,0,1,0,194.7,69.9a8.6,8.6,0,0,1-4-4.2,68,68,0,0,0-125.4,0,8.6,8.6,0,0,1-4,4.2,64,64,0,1,0,62.2,111.3A8.2,8.2,0,0,1,132.5,181.2Z"
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
              />
            </svg>
          </div>
          <h2>
            Origem
          </h2>
          <h3 />
        </div>
        <div>
          <div>
            <svg
              fill="currentColor"
              height="32"
              viewBox="0 0 256 256"
              width="32"
              xmlns="http://www.w3.org/2000/svg"
            >
              <rect
                fill="none"
                height="256"
                width="256"
              />
              <path
                d="M179.4,148A52.2,52.2,0,0,1,136,191.4"
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
              />
              <path
                d="M72,65.4C56.6,88.6,44,115.2,44,140a84,84,0,0,0,168,0c0-52-36-92-65.2-121.1h0L112,92,72,65.4Z"
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="16"
              />
            </svg>
          </div>
          <h2>
            Tipo de torra
          </h2>
          <h3 />
        </div>
      </div>
      <button>
        Adicionar ao carrinho
      </button>
    </div>
    <img
      alt="product"
      src=""
    />
  </div>
</div>
`;
